src/TodoComponents/InputTodo.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+    };
+    this.handleFormInputChange = this.handleFormInputChange.bind(this);
+    this.handleSubmitForm = this.handleSubmitForm.bind(this);
+  }
+
+  handleFormInputChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  handleSubmitForm = (e) => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.handleNewEntry(this.state.title);
+      this.setState({
+        title: '',
+      });
+    } else {
+      alert('You must add a title');
```

Comment:
Great job, @Irirwanirira, following the specifications provided by the follow-through tutorial. However, I believe that creating a separate component for this great UI feature would make your work stand out, as well as it will show your commitment to coding standards.

Reply:
@AbayomiOlaoye 

 Thank you for the review